Week1/Js-exercises/5-round-a-number.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+let z;
+z = 7.25;
+console.log(z);
+
+let a;
+a = Math.round(z);
+console.log(a);
+
+
+let newVariable;
+
+if(z > a){
+    newVariable = z;
+}else {
+    newVariable = a;
+};
+
+console.log(newVariable);
```

Comment:
👍 but you could also explore using conditional operators for assigning value to `newVariable` as this is a good case of using them